src/odemis/acq/test/stream_test.py

```diff
@@ -3172,6 +3109,7 @@ def test_acq_arpol_leech(self):
 
         ars.integrationTime.value = 1  # s
         ars.repetition.value = (2, 3)  # TODO use fixed repetition value -> set ROI?
+        assert ars.repetition.value == (2, 3)
```

Comment:
It happened that the repetition changed after calling self._roiToPhys. Why do you put the assert here?

Reply:
Yes, but here we care that the repetition is the one we've set.